libraries/rushell/src/test/__snapshots__/Tokenizer.test.ts.snap

```diff
@@ -0,0 +1,336 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`00: empty inputs 1`] = `
+Object {
+  "input": "",
+  "tokens": Array [],
+}
+`;
+
+exports[`00: empty inputs 2`] = `
+Object {
+  "input": "[r][n]",
+  "tokens": Array [
+    Array [
+      "NewLine",
+      "[r][n]",
+    ],
```

Comment:
Might be better to use an object with named properties.

Reply:
Good point

---
In reply to: [197313558](https://github.com/Microsoft/web-build-tools/pull/715#discussion_r197313558) [](ancestors = 197313558)